Spanner/src/Admin/Database/V1/Client/DatabaseAdminClient.php

```diff
@@ -370,6 +376,7 @@ public static function parseName(string $formattedName, string $template = null)
      */
     public function __construct(array $options = [])
     {
+        $options = $options + $this->getDefaultEmulatorConfig();
```

Comment:
unimportant nit, but missed opportunity to have made this 
```suggestion
        $options += $this->getDefaultEmulatorConfig();
```

Reply:
:-/